src/SharpGLTF.Cesium/BinaryTable.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SharpGLTF
+{
+    public static class BinaryTable
+    {
+        public static byte[] GetOffsetBuffer(IReadOnlyList<string> strings)
+        {
+            var offsetBuffer = GetOffsets(strings);
+            var offsetBytes = GetBytes(offsetBuffer);
+            return offsetBytes;
+        }
+
+        public static byte[] GetBytes<T>(IReadOnlyList<T> values)
+        {
+            var type = typeof(T);
+            int size = 0;
+            if (type == typeof(float))
+            {
+                size = sizeof(float);
+            }
+            else if (type == typeof(int))
+            {
+                size = sizeof(int);
+            }
+            else if (type == typeof(uint))
+            {
+                size = sizeof(uint);
+            }
```

Comment:
T in the binary table can be scalar ((U)INTX/FLOAT32/FLOAT64), but also Vector, Matrix, Boolean, Enum, string, Array see https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata#binary-table-format

Reply:
oh, if it also contains strings then the inmutable/span tricks are not possible :(

Reply:
I would put this utility class in SharpGLTF.Memory  namespace, it's where all the buffer/array/byte utilities are located

Reply:
you mean moving the BinaryTable class (and test class) to  project SharpGLTF.Core, folder memory? 
The format of the binary table is specific for 3D Tiles (https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata#binary-table-format), though I can imagine it can also be used in other type of projects.

Reply:
No, I mean creating a Memory directory (and namespace) in the Cesium project

Reply:
ok done in https://github.com/vpenades/SharpGLTF/pull/206/commits/a9af095d2c03a2d16275d9f283e46ab4980fd4d6